### DISCUSSION TITLE
7zip extraction running for 10s of hours

### DISCUSSION THREAD

**Discussion by frederikjchristensen** [Comfy-Org/ComfyUI#2436] [Q&A]
Is anyone else having problems installing on windows? The extraction takes forever, i let it run overnight and it still says there is 50 hours left. This is clearly not supposed to happen but i have no clue why it can't extract a simple zip file.

**Comment by monsieurpooh** [comment]
Yes, even on SSD. Really mind-boggling

**Comment by monsieurpooh** [comment]
I used Winrar "Extract to" and it worked quite quickly. Maybe a bug with 7zip and/or windows 11

**Reply by kiralpoon** [reply]
> I used Winrar "Extract to" and it worked quite quickly. Maybe a bug with 7zip and/or windows 11

I had verified this!! And 7zip or windows 11 still cannot extract it efficiently. WINRAR works !!

**Comment by re-ffm** [comment]
@frederikjchristensen same experience for me. Currently at 8h52m with less than an hour to go. I think trying another program may be the solution. I'll report back the results if I end up doing a comparison.

Filename: new_ComfyUI_windows_portable_nvidia_cu121_or_cpu.7z
Compressed size: 1075 MB
Total size: 4842 MB
Speed: 151 KB/s
Version 7Zip 22.01 (x64)

My specs:
Windows 10
AMD Ryzen 9 39000X 12-Core Processor
32 GB RAM
Secondary SSD drive

**Reply by re-ffm** [reply]
@frederikjchristensen I just repeated the extract using WinRAR to compare the performance. It took one minute! I'm never using the 7zip tool again.

**Comment by re-ffm** [comment]
Use WinRAR to extract the 7z file.

**Comment by rmorse** [comment]
Had the same issue (overnight extraction), found a reddit post to disable AV, and yup now it works quickly.

After disabling Avast it completed in about 10 seconds.

**Comment by Xenodermus** [comment]
Ok , I came here to confirm, with WinRar it's much faster and definitely some bug with 7zip which I use for everything else and now thinking of it, some games extraction was painfully long, half a day at times, I thought it was normal because of too many files, now I know better for next time :)
